README.md

```diff
@@ -18,3 +18,44 @@ usage: run_experiment.py [-h] [-j THREADS] experiment_yaml
 run_experiment.py: error: the following arguments are required: experiment_yaml
 ```
 
+**How to Install BFASST** 
+
+* Clone the github repository into your home directory in linux. Repository is found at https://github.com/byuccl/bfasst. Use ```git clone https://github.com/byuccl/bfasst.git.``` (Note: if Git is not installed on your device, use ```sudo apt install git``` first.) 
+
+* Use ```git pull``` to put the updated repo onto your device. 
+
+* Open a terminal in Visual Studio. Use ```cd bfasst``` if you're not in the bfasst folder already. (It should currently say _YOURNAME@____ _:~/bfasst$_)
+
+* Set up your SSH with caedm. You'll know if you set it up correctly when you can use ```ssh caedm``` and log in without having to enter your password. Follow the guide at https://byu-cpe.github.io/ComputingBootCamp/tutorials/linux/ .
+
+* Also make sure to add your ssh key to your github account. Follow the guide at https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account
+
+* Open _scripts/bfasst/config.py_. Change _fsj/squallz_ to your caedm login info. For instance, if you connect to caedm and enter ```pwd``` and your info is _/hij/username_, change _fsj/squallz_ to _hij/username_.
```

Comment:
Could you move this to a CAEDM specific section for those that want to run conformal?